docs/source/markdown/podman-network-create.1.md

```diff
@@ -11,22 +11,22 @@ Create a network configuration for use with Podman. By default, Podman creates a
 A *Macvlan* connection can be created with the *-d macvlan* option. A parent device for macvlan or
 ipvlan can be designated with the *-o parent=`<device>`* or *--network-interface=`<device>`* option.
 
-If no options are provided, Podman will assign a free subnet and name for the network.
+If no options are provided, Podman assigns a free subnet and name for the network.
 
-Upon completion of creating the network, Podman will display the name of the newly added network.
+Upon completion of creating the network, Podman displays the name of the newly added network.
 
-NOTE: The support for the network name pasta is deprecated and will be removed in the next major
+NOTE: The support for the network name "pasta" is deprecated and is removed in the next major
```

Comment:
This is an actual future tense because it will actually happen in the future.

Reply:
good catch!